packages/dialtone-tokens/themes/config.js

```diff
@@ -24,6 +42,38 @@ export function setTheme (theme, rootNode = document.documentElement, contrastTh
   }
 }
 
+/**
+ * Layered theme setter (new optimized system)
+ */
+function _setThemeLayered(theme, rootNode = document.documentElement) {
+  if (rootNode?.shadowRoot) {
+    rootNode = rootNode.shadowRoot;
+  }
+
+  // Load core tokens only once
+  if (theme.core && !coreTokensLoaded) {
+    _setStyleTag('dialtone-css-core', theme.core, rootNode);
+    coreTokensLoaded = true;
+  }
+
+  // Load base colors only once
+  if (theme.baseColors && !rootNode?.querySelector('#dialtone-css-base-colors')) {
```

Comment:
Why do we use coreTokensLoaded to check for existence in the first one, but a querySelector here? Not a big deal, but feels inconsistent.